assets/lokomotive-kubernetes/aws/flatcar-linux/kubernetes/cl/controller.yaml.tmpl

```diff
@@ -7,7 +7,7 @@ systemd:
         - name: 40-etcd-cluster.conf
           contents: |
             [Service]
-            Environment="ETCD_IMAGE_TAG=v3.4.7"
+            Environment="ETCD_IMAGE_TAG=v3.4.9"
```

Comment:
Should we wait for the upstream kubernetes to bump up its etcd version to 3.4.9 before we do ? 
https://github.com/kubernetes/kubernetes/issues/91266

If its something that can wait then lets wait.

Reply:
Sure we can wait.

Reply:
I think it's fine to update, we get fixes quicker :)

Reply:
or bugs :see_no_evil: